flink-python/pyflink/table/table_environment.py

```diff
@@ -1082,6 +1086,13 @@ def _from_elements(self, elements, schema):
         finally:
             os.unlink(temp_file.name)
 
+    def _set_python_executable_for_local_executor(self):
+        jvm = get_gateway().jvm
+        j_config = get_j_env_configuration(self)
+        if not j_config.containsKey(jvm.PythonOptions.PYTHON_EXECUTABLE.key()) \
```

Comment:
Should we consider the case when a user has set the `python.executable` via `table_config.get_configuration()`?

Reply:
Yes, the configuration in `TableConfig` has priority over the configuration in `ExecutionEnvironment`. So even if we set the `python.executable` in `ExecutionEnvironment`, it will not override the user configuration in `TableConfig`.